frontend/src/components/maps/leaflet/LayerPopup/hooks/useLayerQuery.tsx

```diff
@@ -27,6 +27,39 @@ interface IUserLayerQuery {
   findByPin: (pin: string) => Promise<FeatureCollection>;
 }
 
+/**
```

Comment:
split the save logic into a separate method in case the caller already has a parcel layer response that they don't need handled via handleParcelDataLayerResponse.